src/Umbraco.Web/PublishedElementExtensions.cs

```diff
@@ -146,57 +146,6 @@ public static T Value<T>(this IPublishedElement content, string alias, string cu
 
         #endregion
 
```

Comment:
@nul800sebastiaan I've just updated the three conflicting files to keep the branch up to date so you can hopefully merge it in, however I just thought it was worth pointing out this section where it seems this code was missing in the Temp8 branch. So I have also removed it here, do confirm this is correct :)